pkg/cluster/cluster.go

```diff
@@ -1761,35 +1766,39 @@ func (c *Cluster) GetSwitchoverSchedule() string {
 }
 
 // Switchover does a switchover (via Patroni) to a candidate pod
-func (c *Cluster) Switchover(curMaster *v1.Pod, candidate spec.NamespacedName) error {
+func (c *Cluster) Switchover(curMaster *v1.Pod, candidate spec.NamespacedName, inMaintWindow bool) error {
```

Comment:
```suggestion
func (c *Cluster) Switchover(curMaster *v1.Pod, candidate spec.NamespacedName, scheduled bool) error {
```

I think this makes it easier to read that the switchover should be scheduled or not because of the maintenanceWindow. Otherwise this reads to me as Switchover being called during maintenanceWindow = true or false. And interestingly, if the call happens during the window there's no need for schedule and `inMaintWindow` will be false. Counter-intuitive naming.